src/core/types/Token.ts

```diff
@@ -22,9 +21,9 @@ export interface TokenView {
   isZapable: boolean;
   sourceIsZapper: boolean;
   allowancesMap: { [tokenAddress: string]: string };
-}
-
-export interface TokenDynamicData {
-  address: Address;
-  priceUsdc: string;
+  supported: {
+    zapperZapIn?: boolean;
+    zapperZapOut?: boolean;
+    ftmApeZap?: boolean;
+  };
```

Comment:
We should look into exporting from SDK the `zapContractId` so that we dont need to touch this file when new zaps get added in SDK. Maybe just add a TODO for now and we can create a ticket for SDK.

Reply:
Saw the TODO directly in the code where this is needed, so no need to place here again :)